ush/plot_forecast_restart.py

```diff
@@ -50,6 +50,8 @@ def main():
         cartopy.config['data_dir']='/scratch2/NAGAPE/epic/UFS_Land-DA_Dev/inputs/NaturalEarth'
     elif yaml_data['machine']=='orion' or yaml_data['machine']=='hercules':
         cartopy.config['data_dir']='/work/noaa/epic/UFS_Land-DA_Dev/inputs/NaturalEarth'
+    elif yaml_data['machine']=='singularity':
+                cartopy.config['data_dir']='SINGULARITY_WORKING_DIR/land-DA_workflow/fix/NaturalEarth'
```

Comment:
@EdwardSnyder-NOAA, is this indentation correct? It looks indented twice.

Reply:
this has been fixed. Thanks!